src/lib.rs

```diff
@@ -86,11 +86,11 @@ mod up_core_api {
 // Types not used by up_rust, but re-exported to up_rust users, keeping them in their respective submodules
 #[cfg(feature = "udiscovery")]
 pub use up_core_api::udiscovery;
-#[cfg(feature = "usubscription")]
-pub use up_core_api::usubscription;
 #[cfg(feature = "utwin")]
 pub use up_core_api::utwin;
 
+pub mod core;
```

Comment:
maybe we can apply the same pattern to `utwin` and `udiscovery` as well? Otherwise we end up with having the `usubscription` types available in `up_rust::core::subscription` only while we have the types of `utwin` and `udiscovery` available in `up_core` ...

Reply:
Yup, makes sense to me. I'll go move them in a similar manner :+1: